File: 260911/task 4/code.cs
namespace TestCS
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int num1, num2, num3;
            float result;

            Console.Write("Number 1 << ");
            num1 = Convert.ToInt32(Console.ReadLine());
            Console.Write("Number 2 << ");
            num2 = Convert.ToInt32(Console.ReadLine());
            Console.Write("Number 3 << ");
            num3 = Convert.ToInt32(Console.ReadLine());

            result = (num1 + num2 + num3) / 3.0f;

            Console.WriteLine($"\nArithmetic mean {(int)result} [{result}]\n\nPress ENTER to exit...");

            Console.ReadLine();
        }
    }
}

File: 260911/task 4/code.cpp
#include <iostream>
#include <conio.h>

int main() {

    int num1, num2, num3;
    float result;

    std::cout << "Number 1 << "; 
    std::cin >> num1;
    std::cout << "Number 2 << "; 
    std::cin >> num2;
    std::cout << "Number 3 << "; 
    std::cin >> num3;

    result = (num1 + num2 + num3) / 3.0;

    std::cout << "\n\n Arithmetic mean " << (int)result << " [" << result << "]\n\nPress any key to exit...\n";

    _getch();

    return 0;
}